src/modules.js

```diff
@@ -110,7 +110,8 @@ var LibraryManager = {
         'library_nodefs.js',
         'library_sockfs.js',
         'library_workerfs.js',
-        'library_tty.js'
+        'library_tty.js',
+        'library_lz4.js',
```

Comment:
I'll add an `LZ4` flag, I think that's most consistent with our other relevant options. Removing that `init` method, it was a silly idea...